### PR TITLE
Feature/create home page

### PR DESCRIPTION
**Pull request by saikatgpae**
This PR is create to 
Fetch data from API.
Store fetch data inside the store.
Display store.
Filter Data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+## Built With
+
+- Major languages HTML, CSS, JavaScript.
+- Frameworks Bootstrap, React.
+- Technologies used Bootstrap, linters, jest.
+
+
+## Live Demo 
+
+[Live Demo Link]()
```
Comment (AnselemOdims) [inline README.md]: - [ ] Please kindly deploy your project and include a link here

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly highlight a piece of the code that you found interesting or proud of making in your presentation video
- [ ] Please kindly include integration tests using React Testing Library
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (saikatgpae) [reply README.md]: Thank you for your suggestion.
Updated the same.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+- Major languages HTML, CSS, JavaScript.
+- Frameworks Bootstrap, React.
+- Technologies used Bootstrap, linters, jest.
+
+
+## Live Demo 
+
+[Live Demo Link](https://saikatgpae.github.io/react-capstone/)
+
+## Video link
```
Comment (SajjadAhmad14) [inline README.md:19]: Please record a new video and only highlight a piece of the code that you found interesting or proud of making. It's asked to implement by the previous code reviewer.

File: src/component/__test__/Header.test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Details from '../__mock__/Details';
+
```
Comment (SajjadAhmad14) [inline src/component/__test__/Header.test.js:4]: Please add integration tests using React Testing Library as asked to implement by the previous code reviewer.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Saikat,
### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 2 above)
Comment (saikatgpae) [reply README.md:19]: Added the same.

File: src/component/__test__/Header.test.js (hunk 3 above)
Comment (saikatgpae) [reply src/component/__test__/Header.test.js:4]: Thank you for your suggestion added the same.

**Review by KDlamini — APPROVED** [review]
### Hi @saikatgpae 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR description
✔️ App works as expected
✔️ Commit messages are meaningful
✔️ No linter errors

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**